### ISSUE TITLE
how to train a new ldm which can generate 512*512

### ISSUE THREAD

**Issue by markgao-916** [CompVis/latent-diffusion#167]
how to train a new ldm which can generate 512*512? training a new autoencoder is needed?
config.yaml as follow:
model:
  base_learning_rate: 2.0e-06
  target: ldm.models.diffusion.ddpm.LatentDiffusion
  params:
    linear_start: 0.0015
    linear_end: 0.0195
    num_timesteps_cond: 1
    log_every_t: 200
    timesteps: 1000
    first_stage_key: image
    cond_stage_key: class_label
    image_size: 64
    channels: 3
    cond_stage_trainable: false
    concat_mode: false
    monitor: val/loss
    unet_config:
      target: ldm.modules.diffusionmodules.openaimodel.UNetModel
      params:
        image_size: 64
        in_channels: 3
        out_channels: 3
        model_channels: 224
        attention_resolutions:
        - 8
        - 4
        - 2
        num_res_blocks: 2
        channel_mult:
        - 1
        - 2
        - 3
        - 4
        num_head_channels: 32
    first_stage_config:
      target: ldm.models.autoencoder.VQModelInterface
      params:
        embed_dim: 3
        n_embed: 8192
        ddconfig:
          double_z: false
          z_channels: 3
          resolution: 256
          in_channels: 3
          out_ch: 3
          ch: 128
          ch_mult:
          - 1
          - 2
          - 4
          num_res_blocks: 2
          attn_resolutions: []
          dropout: 0.0
        lossconfig:
          target: torch.nn.Identity
    cond_stage_config: __is_unconditional__
data:
  target: main.DataModuleFromConfig
  params:
    batch_size: 42
    num_workers: 5
    wrap: false
    train:
      target: ldm.data.faceshq.FFHQTrain
      params:
        size: 512
    validation:
      target: ldm.data.faceshq.FFHQValidation
      params:
        size: 512

**Comment by Guanys-dar** [comment]
I am wondering the corrected config file.

**Comment by YoussefNawar** [comment]
Did you find the correct config?

**Comment by Guanys-dar** [comment]
Nope. Maybe we can follow the published version in HuggingFace.

**Comment by TorAP** [comment]
Solved this?

**Comment by YoussefNawar** [comment]
> Solved this?

Yes I trained an Auto encoder to generate 512x512 images from the kl-16 config

**Comment by TorAP** [comment]
Can I use any autoencoder when training the ldm? E.g., if I train to output dims 648x890 (or at least this ratio) can I then simply use this for the ldn training?

**Comment by iris0329** [comment]
Hi @YoussefNawar , may I ask how could you decide to use kl-16 config rather than other config like kl-8 config

**Comment by YoussefNawar** [comment]
I tried both the kl-16 and kl-8, the kl-8 performed slightly better so I went with it. Overall, I don't think it would make much of a difference.

**Comment by iris0329** [comment]
I see, thank you!
